### PR TITLE
made contact form functional

### PR DESCRIPTION
**Pull request by sethharling**
-simplified form
-once submitted, form will send email containing information
-uses node server on localhost:5000

### CODE REVIEW

File: web/src/Index.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { render } from 'react-dom';
-import './index.css';
+import './Index.css';
```
Comment (sethharling) [inline web/src/Index.jsx]: This was made in order to get the local server running on my computer. I'm not sure if this will cause errors for others

Comment (chillyssa) [reply web/src/Index.jsx]: I had that same issue! Thanks for updating here.

**Comment by chillyssa** [comment]
@sethharling I merged main into this PR because there were some changes that affected this pr.

**Review by chillyssa — APPROVED** [review]
This is coming along great, Seth! Thanks for getting this in! Interesting about the deprecation on nodemailer! Glad we are able to have it for demo at least!

**Review by Jesusxx-240 — APPROVED** [review]
Good job.

**Pull request merged**